docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+    "firstname": "String",
+    "lastname": "String",
+    "age": INT,
+    "gender": "M or F",
+    "address": "String",
+    "employer": "String",
+    "email": "String",
+    "city": "String",
+    "state": "String"
+}
+
+After downloading the data sets, load them into Elasticsearch with the following commands:
+
+[source,shell]
+$ curl -XPOST 'localhost:9200/bank/account/_bulk?pretty' --data-binary @accounts.json
+$ curl -XPOST 'localhost:9200/play/shakespeare/_bulk?pretty' --data-binary @shakespeare.json
+
+These commands may take some time to execute, depending on the computing resources available.
+
+Verify successful loading with the following command:
+
+[source,shell]
+curl 'localhost:9200/_cat/indices?v'
+
+You should see output similar to the following:
+
+[source,shell]
+health status index               pri rep docs.count docs.deleted store.size pri.store.size
+yellow open   bank                  5   1       1000            0    418.2kb        418.2kb
+yellow open   shakespeare           5   1     111396            0     17.6mb         17.6mb
+
+[float]
+[[tutorial-define-index]]
+=== Defining Your Index Patterns
+
+Each set of data loaded to Elasticsearch has an https://www.elastic.co/guide/en/kibana/current/settings.html#settings-create-pattern[index pattern]. In the previous section, the Shakespeare data set has an index named `shakespeare`, and the accounts 
+data set has an index named `bank`. An _index pattern_ is a regular expression that can 
+match multiple indices. For example, in the common logging use case, a typical index name contains the date in MM-DD-YYYY 
+format, and an index pattern for May would look something like `logstash-05-*`.
+
+For this tutorial, any pattern that matches either of the two indices we've loaded will work. Open a browser and 
+navigate to `localhost:5601`. Click the *Settings* tab, then the *Indices* tab. Click *Add New* to define a new index 
+pattern. Since these data sets don't contain time-series data, make sure the *Index contains time-based events* box is 
+unchecked. Specify `shakes*`  as the index pattern for the Shakespeare data set and click *Create* to define the index 
+pattern, then define a second index pattern named `ba*`.
+
+[float]
+[[tutorial-discovering]]
+=== Discovering Your Data
+
+Click the *Discover* tab to display Kibana's data discovery functions:
+
+image::images/tutorial-discover.png[]
+
+Right under the tab itself, there is a search box where you can search your data. Searches take a specific 
+{ref}/query-dsl-query-string-query.html#query-string-syntax[query syntax] that enable you to create custom searches, 
+which you can save and load by clicking the buttons to the right of the search box.
+
+Beneath the search box, the current index pattern is displayed in a drop-down. You can change the index pattern by 
+selecting a different pattern from the drop-down selector.
+
+Try selecting the `ba*` index pattern and putting the following search into the search box:
```

Comment:
A bit more on query syntax would be good, and maybe a short plain-english explanation of the query.